src/main/java/com/gorlah/kappabot/rest/model/CommandRequest.java

```diff
@@ -1,12 +1,33 @@
 package com.gorlah.kappabot.rest.model;
 
 import com.gorlah.kappabot.function.BotRequestMetadata;
+import com.gorlah.kappabot.integration.discord.DiscordIntegration;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Value;
 
+import javax.validation.constraints.NotEmpty;
+
 @Value
+@Builder
+@AllArgsConstructor
+@Schema(example = """
+        {
+          "author": "Gorlah",
+          "message": "hello"
+        }
+        """)
```

Comment:
It could, but it would be slightly misleading since it's going to use the default if you don't have a discord integration turned on. I think the default example should represent functionality that works right out of the box. You can see the full schema, which shows the optional source field, by clicking from `Example Value` to `Schema`. 

![firefox_busl54snYS](https://user-images.githubusercontent.com/5797900/93694828-0ca05180-fad6-11ea-9458-9f07cd3901b7.png)

Reply:
Yeah it works without specifying a source now, but I thought that was considered a fallback behavior. I considered the purpose of this endpoint as a way to debug handling of a real source.

Reply:
I guess it just depends on how you look at it. I know it has never really turned into this, but I always tried to make this version of KappaBot as extensible as possible. Starting with HipChat, Discord, and (eventually?) Teams, I've rewritten the project basically 3 different times. The core of the command processing remained the same, it was just the sources that changed. So as applications come and go, I think it makes sense to just have a "default" behavior that integrates to nothing. 

If I need to test Discord I can add `"source": "discord"`, or also OpenAPI supports multiple examples so I could technically add a default and Discord example.